ui/raidboss/data/07-dt/raid/r7n.txt

```diff
@@ -1,9 +1,274 @@
+140.2 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+143.2 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+147.2 "Stoneringer: The Bludgeoning" Ability { id: "A532", source: "Brute Abombinator" }
+156.1 "Brutish Swing" Ability { id: "A534", source: "Brute Abombinator" }
+157.1 "Brutish Swing" Ability { id: "A535", source: "Brute Abombinator" }
+158.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+163.1 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+173.0 "Brutish Swing" Ability { id: "A539", source: "Brute Abombinator" }
+175.9 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+177.1 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+181.7 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+182.9 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
+183.0 "Electrogenetic Force" Ability { id: "A547", source: "Brute Abombinator" }
+190.0 "Abominable Blink" Ability { id: "A542", source: "Brute Abombinator" }
+191.0 "Abominable Blink" Ability { id: "A892", source: "Brute Abombinator" }
+192.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+199.0 "Revenge of the Vines" Ability { id: "A543", source: "Brute Abombinator" }
+201.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+206.0 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+216.9 "Sporesplosion" Ability { id: "A544", source: "Brute Abombinator" }
+223.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.4 "Brutish Swing" Ability { id: "A53D", source: "Brute Abombinator" }
+227.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+228.3 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+229.5 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+234.1 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+235.3 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
+235.4 "Electrogenetic Force" Ability { id: "A547", source: "Brute Abombinator" }
+239.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+242.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+248.2 "Powerslam" Ability { id: "A548", source: "Brute Abombinator" }
+
+# Third arena
+269.8 "Stoneringer 2: Stoneringers" Ability { id: "A54A", source: "Brute Abombinator" }
+278.7 "Spore Sac" Ability { id: "A529", source: "Brute Abombinator" }
+280.8 "--sync--" Ability { id: "A516", source: "Brute Abombinator" }
+283.8 "Spore Sac" Ability { id: "A52A", source: "Brute Abombinator" }
+286.4 "Brutish Swing" Ability { id: "A54C", source: "Brute Abombinator" }
+289.2 "Brutish Swing" Ability { id: "A550", source: "Brute Abombinator" }
+290.2 "Brutish Swing" Ability { id: "A54F", source: "Brute Abombinator" }
```

Comment:
```suggestion
286.4 "Brutish Swing" Ability { id: "A54C", source: "Brute Abombinator" }
289.2 "Brutish Swing" Ability { id: ["A54E", "A550", "A63A", "A63B"], source: "Brute Abombinator" }
290.2 "Brutish Swing" Ability { id: "A54F", source: "Brute Abombinator" }
```

Here and elsewhere, the "sandwich" of Brutish Swings ending in `A54F` always includes one of the four alternatives. Unfortunately I don't have video from any of my normal mode clears, so I'll have to record that some other time and figure out what's going on with the first and third abilities. I don't want to hold the PR up over my lack of data, but could you add a TODO note for Brutish Swing?